packages/analytics-core/src/types/network-tracking.ts

```diff
@@ -45,6 +45,17 @@ export interface NetworkCaptureRule {
    * @defaultValue `["*"]` all hosts (except amplitude)
    */
   hosts?: string[];
+  /**
+   * URL patterns to allow for network capture. Supports wildcard.
+   * @experimental This feature is experimental and may not be stable
+   * @defaultValue `["*"]` all URLs
+   */
+  urls?: (string | RegExp)[];
```

Comment:
Can we mention that urls takes precedence over hosts and if urls is set, hosts is ignored?